tests/utils/test_file_utils.py

```diff
@@ -85,29 +95,39 @@ def test_context_managers_two_context(self, mock_stdout):
         # The output should be wrapped with an English and French welcome and goodbye
         self.assertEqual(mock_stdout.getvalue(), "Bonjour!\nWelcome!\nTransformers are awesome!\nBye!\nAu revoir!\n")
 
-    def test_find_labels(self):
```

Comment:
In this test:
- moved imports to the top
- separated it into 3 tests (one per framework)
- added a check to confirm that it works regardless of the class name